app/controllers/home_controller.rb

```diff
@@ -1,3 +1,8 @@
 class HomeController < ApplicationController
   before_action :authenticate_customer!
+
+  def index
```

Comment:
Ransack applying on product should not be in products_controller?

Reply:
It exists there too, why we are not using products controller?

Reply:
Asad Bhai, I'm already using products_controller.